lib/services/local/access_list.go

```diff
@@ -223,5 +224,5 @@ func (a *AccessListService) DeleteAllAccessListMembers(ctx context.Context) erro
 }
 
 func lockName(accessListName string) string {
-	return string(backend.Key("access_list", accessListName))
+	return strings.Join([]string{"access_list", accessListName}, string(backend.Separator))
```

Comment:
I'm pretty sure that this has been fixed already https://github.com/gravitational/teleport/pull/31279

Reply:
Yep, just had to rebase.